components/script/script_runtime.rs

```diff
@@ -831,47 +832,67 @@ impl StreamConsumer {
     }
 }
 
+/// Implements the steps to compile webassembly response mentioned here
+/// <https://webassembly.github.io/spec/web-api/#compile-a-potential-webassembly-response
```

Comment:
nit: add a trailing > so it shows up as a link in the generated documentation.

Reply:
Done